resources/tiers.md

```diff
@@ -0,0 +1,84 @@
+# SDK tiers
+
+The objective of Meilisearch's Integrations Team is to **ease Meilisearch usage to as many developers as possible**.
+
+Given this statement, **tiers** are nothing more than a __group of SDKs/integrations__ where the integrations team can fairly divide their attention. Unfortunately is not possible to keep all the SDKs 100% updated against the new Meilisearch releases all the time, the Integrations Team is a polyglot small team that maintains 30+ different integrations and tools, so it is humanly impossible to consistently meet the Meilisearch quality standards.
+
+The ultimate goal of introducing tiers is to allow the team to have more time for new projects/integrations so that more time can be invested and more users will be impacted.
+
+It's been using [data collected here](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html) and data like GitHub stars, forks, and watchers to measure the impact of an SDK.
+
+It is worth remembering that this priority is just a concept introduced by the team where it aims to give a decision weight before
+we invest time in some issues of some SDK in question.
```

Comment:
```suggestion
It is worth remembering that this priority list is just a concept introduced by the team aiming to weigh a task before investing time on it. 
```

Not sure I understand the sentence correctly.

Reply:
It means nothing is written in stone. It does not mean that we never solve an issue in a `tier #3` SDK if we have an open issue in `tier #2` for example.

The idea here is to give the reader the sense that we're going to use this concept for prioritization purposes and not to block the road for impactless SDKs. 

It is clearer now? Maybe I should change the way I wrote it... Do you have any suggestion now?

Reply:
WDYT?

> It is worth remembering that this priority is just a concept introduced by the team that aims to give a decision weight before investing time in some issues of the SDKs. So it does not mean we will never work on feature requests from the bottom tiers anymore, but it says we will prioritize the top tiers most of the time transparently.

Reply:
Good for me!